docs/enhancements/backup-restore-support.md

```diff
@@ -0,0 +1,151 @@
+---
+title: backup-restore-support
+authors:
+  - "@danielerez"
+creation-date: 2024-08-08
+last-updated: 2024-08-08
+---
+
+# Backup/Restore Support
+
+## Summary
+
+The Assisted Installer should support backup/restore and disaster recovery scenarios, either using OADP (OpenShift API for Data Protection) for ACM (Advanced Cluster Management), or, using ZTP (Zero Touch Provisioning) flows.
+I.e. the assisted-service should be resilient in such scenarios which, for this context and effort, means that restored/moved spoke clusters should keep the same state and behave the same on the new hub cluster. 
+
+## Motivation
+
+Provide resiliency in the assisted-service for safe backup/restore flows, allowing spoke clusters to be used without any restriction after DR scenarios or moving between hubs.
+
+### Goals
+
+Spoke clusters with BMHs should function properly and have a consistent state after these scenarios: 
+* Spoke clusters restored on a passive hub with ACM/OADP functionality.
+* Spoke clusters moved between hubs using ZTP flows.
+
+### Non-Goals
+
+* Improve import day2 cluster solution by getting full host data (e.g. using an agent on existing nodes).
+
+## Proposal
+
+### User Stories
```

Comment:
All stories are "as an ACM user".  Why do we care about how resources are backed up and restored?  OADP, ArgoCD, or typing each CRD by hand should be the same, right?

I think the important variations are:
1. Are they using BMHs?  This affects installed hosts getting rebooted with our ISO (bad) and uninstalled hosts getting rebooted with our ISO (good, so we can use them in the new hub).  If they aren't using BMHs, they need to manually boot uninstalled hosts if the hub changed.
2. What is the cluster state?  Installation not started, installing, or installed.